flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/operators/windowing/functions/InternalWindowFunction.java

```diff
@@ -39,5 +40,31 @@
 	 * @param out    A collector for emitting elements.
 	 * @throws Exception The function may throw exceptions to fail the program and trigger recovery.
 	 */
+	@Deprecated
 	void apply(KEY key, W window, IN input, Collector<OUT> out) throws Exception;
```

Comment:
I noticed an issue when removing apply. The method is used inside of AccumulatingKeyedTimePanes which takes in an AbstractStreamOperator as an argument to its evaluateWindow method. When creating the context I can get the global keyed state backend from the operator, but not the partitioned state because those methods are protected. Now the only two uses of this class are its subclasses which have both been deprecated. My question is, do you think I should modify the evaluateWindow method to accept a keyed state store which wraps the operator partitioned state or just throw an exception on context.windowState() because all valid uses of this method have been deprecated?